components/TaskCard/index.js

```diff
@@ -1,25 +1,31 @@
 import { useState } from "react";
 
-import { Tasks, Task, TaskListItem } from "./TaskCard.styled";
+import { Tasks, Task, TaskListItem, Label, Labels } from "./TaskCard.styled";
 
 export const TaskCard = ({ project, index }) => {
   const [visible, setVisible] = useState(false);
   const projectClickHandler = () => {
     setVisible(!visible);
   };
+  console.log(project, index);
+  const { projectTitle, tasks, projectDue, projectSize } = project;
   return (
     <TaskListItem onClick={projectClickHandler}>

```

Comment:
This is a very good idea! Maybe there could be an info that you can expand the card? Maybe with: "click for more information about your project". if we start from the dumbest users i could imagine that the info would be helpful.

Reply:
Thank you, well noticed. I will do it in the next US.